javascript-loops-practice/get-indexes.js

```diff
@@ -0,0 +1,8 @@
+/* exported getIndexes */
+function getIndexes(array) {
+  var array1 = [];
```

Comment:
Why `array1`? That name doesn't carry any real meaning. How about something like `output`?